client/assets/pages/_event_details-page.scss

```diff
@@ -72,6 +72,33 @@
       cursor: not-allowed;
     }
   }
+  &__slack_button {
+    margin: rem(14px) 0 0 rem(23px);
+    background-color: #3f0f3f;
+    font-size: 1.25em;
+    color: white;
+    width: rem(180px);
+    height: rem(41px);
+  }
+  &__menu {
+    position: absolute;
+    width: rem(180px);
+    margin: 0 0 0 rem(210px);
+    top: 180px;
+    box-shadow: 1px 1px 15px rgba(0, 0, 0, 0.1);
+  }
+  &__menu a {
```

Comment:
It would be nice for us to take advantage of Sass here and follow the predefined BEM styling convention. You can nest related styles like: 
```
.menu {
    ....
    &__item {
        ....
    }
}
```

Reply:
okay... got it.
Thanks

Reply:
The nest looks good now.